edge-util/src/Microsoft.Azure.Devices.Edge.Util/metrics/Metrics.cs

```diff
@@ -12,7 +12,7 @@ public static class Metrics
 
         public static IMetricsListener Listener { get; private set; } = new NullMetricsListener();
 
-        public static void Init(IMetricsProvider metricsProvider, IMetricsListener metricsListener, ILogger logger)
+        public static void InitWithServer(IMetricsProvider metricsProvider, IMetricsListener metricsListener, ILogger logger)
```

Comment:
I didn't find any usage of this method.

Reply:
It's not used now since edgeAgent uses dependency injection instead of the global variables. If we want another module to use this library the same way as edgeHub, it would use this if it doesn;t have asp.net